artifactory/cli.go

```diff
@@ -2834,6 +2907,185 @@ func permissionTargetDeleteCmd(c *cli.Context) error {
 	return commands.Exec(permissionTargetDeleteCmd)
 }
 
+func usersCreateCmd(c *cli.Context) error {
+	if show, err := showCmdHelpIfNeeded(c); show || err != nil {
+		return err
+	}
+
+	if c.NArg() != 0 {
+		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
+	}
+
+	rtDetails, err := createArtifactoryDetailsByFlags(c, false)
+	if err != nil {
+		return err
+	}
+
+	usersCreateCmd := usersmanagement.NewUsersCreateCommand()
+	csvFilePath := c.String("csv")
+	if csvFilePath == "" {
+		return cliutils.PrintHelpAndReturnError("missing --csv <File Path>", c)
+	}
+	usersList, err := createUsersListFromCSV(csvFilePath)
+	if err != nil {
+		return err
+	}
+	if len(usersList) < 1 {
+		return fmt.Errorf("An empty input file was given.")
+	}
+	// Run command.
+	usersCreateCmd.SetRtDetails(rtDetails).SetUsers(usersList).SetUsersGroups(strings.Split(c.String("user-groups"), ",")).SetReplaceExistUsersFlag(c.Bool("replace-exist"))
+	return commands.Exec(usersCreateCmd)
+}
+
+func usersDeleteCmd(c *cli.Context) error {
```

Comment:
Let's ask the user to confirm before deleting, unless --quite is sent or we're in CI mode. There's already code that can verifies both.
The same goes for the group delete command.